js/main.js

```diff
@@ -1,2 +1,31 @@
 /* global data */
 /* exported data */
+const $form = document.querySelector('#entry-form');
+$form.addEventListener('submit', handleSubmitCreate);
+const $photoInput = document.querySelector('#photo');
+$photoInput.addEventListener('input', handlePhotoChange);
+const $image = document.querySelector('.img-preview');
+imgError(); // to escape the lint
+
+function handleSubmitCreate(e) {
+  event.preventDefault();
+  const input = event.target.elements;
+  const newEntry = {
+    id: data.nextEntryId,
+    title: input.title.value,
+    photo: input.photo.value,
+    notes: input.notes.value
+  };
+  data.entries.push(newEntry);
+  const dataJSON = JSON.stringify(data);
+  localStorage.setItem('data-local-storage', dataJSON);
```

Comment:
local storage functionality should only be done when the page unloads, you don't want to directly access it here.